templates/crate/details.html

```diff
@@ -105,21 +101,7 @@
                             <div class="pure-menu pure-menu-scrollable sub-menu">
                                 <ul class="pure-menu-list">
                                     {# List all dependencies that the current release has #}
-                                    {%- if details.dependencies -%}
-                                        {%- for dep in details.dependencies -%}
-                                            <li class="pure-menu-item">
-                                                <a href="/crate/{{ dep[0] }}/{{ dep[1] }}" class="pure-menu-link">
-                                                    {{ dep[0] }} {{ dep[1] }}
-                                                    <i class="dependencies {{ dep[2] | default(value='') }}">{{ dep[2] | default(value="") }}</i>
-                                                    {%- if dep[3] -%}
-                                                        <i> optional </i>
-                                                    {%- endif -%}
-                                                </a>
-                                            </li>
-                                        {%- endfor -%}
-                                    {%- else -%}
-                                        &mdash;
-                                    {%- endif -%}
+                                    {% call macros::dependencies_list(link_prefix="/crate", dependencies=details.dependencies, if_empty="&mdash;") %}
```

Comment:
this seems to show all dependencies mistakenly as optional.

Reply:
Yes indeed.